include/alpaka/kernel/Traits.hpp

```diff
@@ -217,7 +217,7 @@ namespace alpaka
                     TArgs...>::createTaskKernel(
                         workDiv,
                         kernelFnObj,
-                        args...);
+                        std::forward<TArgs>(args)...);
```

Comment:
The method return type is still mising the `std::forward`. (C++14 will allow us to remove it completely)